client/blocks/post-item/index.jsx

```diff
@@ -18,13 +18,14 @@ import { getEditorPath } from 'state/ui/editor/selectors';
 import { getSelectedSiteId } from 'state/ui/selectors';
 import { getNormalizedPost } from 'state/posts/selectors';
 import { isSingleUserSite } from 'state/sites/selectors';
-import { areAllSitesSingleUser } from 'state/selectors';
+import { areAllSitesSingleUser, canCurrentUserEditPost } from 'state/selectors';
 import {
 	isSharePanelOpen,
 	isMultiSelectEnabled,
 	isPostSelected,
 } from 'state/ui/post-type-list/selectors';
 import { hideSharePanel, togglePostSelection } from 'state/ui/post-type-list/actions';
+import ExternalLink from 'components/external-link';
```

Comment:
I wasn't aware of this component, nice catch.  I see that it [sets an appropriate `rel` attribute](https://github.com/Automattic/wp-calypso/blob/77fa51735964a8d8c1f86937edbeb0a52ac5abde/client/components/external-link/index.jsx#L50-L52) for us.